website/_enemies/forest/totem-small.md

```diff
@@ -12,4 +12,6 @@ hp: 3
 game_name: enemy_stonebro
 ---
 
-Spins around - don't touch its hitbox! Knock it back with a light attack to keep the distance.
+Has an active hitbox as long as it is moving above a certain speed. Can be knocked back with attacks.
```

Comment:
OP said he doesn't know on Discord, so I'm marking as resolved as it is indeed hard to measure.

Reply:
reopened because I think I have an idea on how to measure this

Reply:
Discovered using MSG's scripting library
```suggestion
Has an active hitbox as long as it is moving at a speed of 400 units per second or higher.

When grounded, it will accelerate up to its max speed at 16.666 u/s/s, unless it is perfectly motionless, then it will accelerate at 26.666 u/s/s for a single frame. Its ground speed is capped at 400 u/s.

When airborne, it will accelerate due to gravity at 16.666 u/s/s.

When colliding with a wall, it will perform an elastic collision, reflecting its speed vector according to the angle of the wall.
```